ui/src/shared/utils/timeMachine.ts

```diff
@@ -19,9 +20,11 @@ import {TimeMachineState} from 'src/shared/utils/TimeMachineContainer'
 export function initialStateFromCell(
   cell: Cell | NewDefaultCell
 ): Partial<TimeMachineState> {
+  const queryDrafts = initialQueryDrafts(cell)
   const initialState: Partial<TimeMachineState> = {
-    queryDrafts: initialQueryDrafts(cell),
+    queryDrafts,
     type: cell.type,
+    queryType: getDeep<QueryType>(cell, 'queries.0.type', QueryType.InfluxQL),
```

Comment:
nitpick `queryDrafts` above is so much more readable, maybe we can take out queryType as well?